File: BAEKJOON/java/9076_점수_집계.java
import java.io.*;
import java.util.ArrayList;
import java.util.Collections;
import java.util.List;
import java.util.StringTokenizer;

public class Main {
    public static List<Integer> score = new ArrayList<>();

    public static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    public static BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

    public static void main(String[] args) throws IOException {
        int tc = Integer.parseInt(br.readLine());
        for (int i = 0; i < tc; i++) {
            input();
            solution();
        }

        bw.flush();
        bw.close();
    }

    private static void input() throws IOException {
        score.clear();
        StringTokenizer st = new StringTokenizer(br.readLine());

        for (int i = 0; i < 5; i++) {
            int a = Integer.parseInt(st.nextToken());
            score.add(a);
        }
    }

    private static void solution() throws IOException {
        Collections.sort(score);

        if (score.get(score.size() - 2) - score.get(1) >= 4) {
            bw.write("KIN\n");
            return;
        }

        int sum = 0;
        for (int i = 1; i < score.size() - 1; i++) {
            sum += score.get(i);
        }

        bw.write(sum + "\n");
    }
}

File: BAEKJOON/cpp/9076_점수_집계.cpp
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;

vector<int> score;

void input() {
	score.clear();

	int a;
	for (int i = 0; i < 5; i++) {
		cin >> a;
		score.push_back(a);
	}
}

void solution() {
	sort(score.begin(), score.end());

	if (score[score.size() - 2] - score[1] >= 4) {
		cout << "KIN" << "\n";
		return;
	}

	int sum = 0;
	for (int i = 1; i < score.size() - 1; i++) {
		sum += score[i];
	}

	cout << sum << "\n";
}

int main()
{
	ios::sync_with_stdio(false);
	cin.tie(NULL);
	cout.tie(NULL);

	int tc;
	cin >> tc;
	for (int i = 0; i < tc; i++) {
		input();
		solution();
	}

	return 0;
}